### PR TITLE
feat-added quiz home page

### PR DESCRIPTION
**Pull request by manish591**
### Added home page for quiz project

### CODE REVIEW

**Review by romabulani — APPROVED** [review]
Looks good to merge

File: css/header.css [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+/* Media Queries */
+/* ------------- */
+
+@media (max-width: 299px) {
+    .hero__action {
+        block-size: 40px;
+    }
+
+    .material-black {
+        font-size: 1.5rem !important;
```
Comment (OmKakatkar) [inline css/header.css:119]: Try not use use `!important` in your code.

File: css/header.css [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+.hero__heading {
+    line-height: 1.2;
+    font-size: var(--fs-700);
+    letter-spacing: 1.2px;
+    max-inline-size: 30ch;
+    margin-inline: auto;
+}
+
+.hero__text {
+    font-size: 1.2rem;
```
Comment (OmKakatkar) [inline css/header.css:103]: Try to be consistent. Either use variables for font-sizes everywhere or don't use at all.

**Review by OmKakatkar — APPROVED** [review]
The code looks good to merge other than the changes mentioned.

File: css/header.css (hunk 1 above)
Comment (manish591) [reply css/header.css:119]: It was a material icon so I have to use important otherwise it's not working.

File: css/header.css (hunk 2 above)
Comment (manish591) [reply css/header.css:103]: ok

**Pull request merged**